spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
 RSpec.describe PushmiPullyu::SwiftDepositer do
-  it 'not valid with nil arguments' do
+  it 'valid connection established' do
     expect { described_class.new(nil, nil) }.to raise_error(RuntimeError)
-  end
 
-  it 'valid connection established' do
```

Comment:
Theses were the wrong two tests to merge haha. The first one (not valid with nil arguments) is fine. It's the other two as your testing the same thing twice.